cpp_api/Makefile

```diff
@@ -20,30 +20,29 @@ EXS_TARGET := false
 DOCKER_TARGET := false
 
 # uncomment to disable generate recursing into Makefiles of subdirectories
-#RECURSE_SUBMAKEFILES := false
+RECURSE_SUBMAKEFILES := false
 
-# standard rules generation makefile
+#standard rules generation makefile
```

Comment:
It would be ideal if we can minimize changes to this `Makefile` to only include ones that change the functionality required.